StormyHares/BangazonApi
# comments

## Ticket Link:
  - Definition of Done

## Description:
  - Commented code and changed linq statements in Product Controller to include 

## Steps to Test:

  - Check GET method on product 
  - Check GET/{ID} method on product
  - Customer data should be joined in and no nulled foregin keys
